### ISSUE TITLE
Storyshots 6.4.1 + axeTest Broken

### ISSUE THREAD

**Issue by m7kvqbe1** [storybookjs/storybook#16834]
**Describe the bug**
Our dependabot bumped `@storybook/addon-storyshots` from `6.3.12` to `6.4.1`. 

Our `axeTest` a11y checks are now failing with a TypeError (see log output below).

```
info => Output directory: /Users/tomhumphris/Projects/mod-uk-design-system/packages/react-component-library/.static_storybook
 FAIL  src/a11y/index.test.tsx
  ● Test suite failed to run

    TypeError: Cannot read properties of undefined (reading 'then')

       5 | const staticStorybookPath = path.join(__dirname, '../../.static_storybook')
       6 |
    >  7 | initStoryshots({
         | ^
       8 |   suite: 'A11y checks',
       9 |   test: axeTest({ storybookUrl: `file://${staticStorybookPath}` }),
      10 | })

      at testStorySnapshots (../../node_modules/@storybook/addon-storyshots/dist/ts3.9/api/index.js:54:66)
      at Object.<anonymous> (src/a11y/index.test.tsx:7:1)
      at TestScheduler.scheduleTests (../../node_modules/@jest/core/build/TestScheduler.js:333:13)
      at runJest (../../node_modules/@jest/core/build/runJest.js:401:19)
      at _run10000 (../../node_modules/@jest/core/build/cli/index.js:320:7)
      at runCLI (../../node_modules/@jest/core/build/cli/index.js:173:3)

Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        2.223 s
Ran all test suites matching /.\/src\/a11y\/index.test.tsx/i.
```

**To Reproduce**

Follow documentation here to setup `axeTest` with addon-storyshots-puppeteer:

https://storybook.js.org/addons/@storybook/addon-storyshots-puppeteer

**System**
```
Environment Info:

  System:
    OS: macOS 12.0.1
    CPU: (10) arm64 Apple M1 Pro
  Binaries:
    Node: 16.13.0 - ~/.nvm/versions/node/v16.13.0/bin/node
    Yarn: 1.22.17 - ~/.nvm/versions/node/v16.13.0/bin/yarn
    npm: 8.1.2 - ~/.nvm/versions/node/v16.13.0/bin/npm
  Browsers:
    Chrome: 96.0.4664.55
    Firefox: 94.0.2
    Safari: 15.1
  npmPackages:
    @storybook/addons: ^6.3.12 => 6.4.1 
```

**Additional context**

https://github.com/defencedigital/mod-uk-design-system/blob/master/packages/react-component-library/src/a11y/index.test.tsx

**Comment by m7kvqbe1** [comment]
Closing because i'm an idiot and didn't update other companion deps.
